pkgs/development/node-packages/node-packages-v10.json

```diff
@@ -16,6 +16,7 @@
 , "browserify"
 , "castnow"
 , "clean-css"
+, "coc-prettier"
```

Comment:
shouldn't that cause a change in pkgs/development/node-packages/node-packages-v10.nix, instead it generates a new file, which seems weird to me.

Reply:
Just as a reminder from November, I agree that this is weird, but apparently regenerating the entire node-packages-v10.nix and updating a ton of NPM packages in the process is the blessed path. I'm basing this off of 1) a conversation I had in the #nixos IRC channel and 2) the documentation.